certified-connectors/FactSet/apiDefinition.swagger.json

```diff
@@ -1260,144 +1387,70 @@
                         "type": "string",
-                        "description": "text"
-                      },
-                      "startChar": {
-                        "type": "integer",
-                        "format": "int32",
-                        "description": "startChar"
-                      },
-                      "endChar": {
-                        "type": "integer",
-                        "format": "int32",
-                        "description": "endChar"
+                        "description": "The type of identifier inputted in the request"
                       },
-                      "lookupText": {
+                      "name": {
                         "type": "string",
-                        "description": "lookupText"
+                        "description": "Name of the requested identifier"
                       },
-                      "entityId": {
+					  "frefListingExchange": {
                         "type": "string",
-                        "description": "entityId"
+                        "description": "The 3 digit fref exchange code for the primary exchange of the security"
                       },
-                      "lookupUrl": {
+					  "< * >": {
                         "type": "string",
-                        "description": "lookupUrl"
+                        "description": "The Key in the Key-Value pair is the name of the requested outputSymbolType. The Value in the Key-Value is the requestId translated to the outputSymbolType."
                       },
-                      "associatedOrgs": {
-                        "type": "array",
-                        "items": {
-                          "type": "object",
-                          "properties": {
-                            "name": {
-                              "type": "string",
-                              "description": "name"
-                            },
-                            "entityId": {
-                              "type": "string",
-                              "description": "entityId"
-                            },
-                            "lookupUrl": {
-                              "type": "string",
-                              "description": "lookupUrl"
-                            }
-                          }
-                        },
-                        "description": "associatedOrgs"
-                      }
                     }
                   },
-                  "description": "entities"
+                  "description": "Response object for Identifier Resolution endpoint."
                 }
               }
             }
           }
         },
-        "summary": "NER - Entities",
-        "operationId": "ner-entities",
+        "summary": "Symbology API",
+        "description": "Translate Entity ID and market security symbols into FactSet Permanent Security Identifiers.",
+        "operationId": "get-symbology",
```

Comment:
@KyleVErwin all operationIds should be in Camel Case. Please change to "GetSymbology". Please fix all operationIds in the swagger file.